packages/commons-server/src/libs/templating-helpers/helpers/hexColor.ts

```diff
@@ -1,10 +1,12 @@
-import { faker } from '@faker-js/faker';
+import { localFaker as faker } from '../../faker';
 
 const hexColor = function () {
   return Math.floor(
-    faker.datatype.number({ min: 0, max: 1, precision: Math.pow(10, -16) }) *
+    faker.number.float({ min: 0, max: 1, precision: Math.pow(10, -16) }) *
       16777215
-  ).toString(16);
+  )
+    .toString(16)
+    .padStart(6, '0');
```

Comment:
Maybe this could be replaced by `faker.color.rgb({ format: 'hex', casing: 'lower' })`?

Reply:
Changed. Will be available in next commit.